sample-hacep-project/sample-hacep-project-client/src/main/resources/configuration.properties

```diff
@@ -0,0 +1,6 @@
+ssl.keystore.location=src/main/resources/keystore.jks
+ssl.truststore.location=src/main/resources/keystore.jks
+ssl.keystore.password=password
+ssl.truststore.password=password
+bootstrap.servers=my-cluster-kafka-bootstrap-hacep.apps-crc.testing:443
```

Comment:
I think it would be better to use some placeholder here instead of URL for crc. In other Openhists this value might be different.

Reply:
Hi @jiripetrlik thank you for the review, I followed your suggestion

Reply:
Thank you! Looks good to me.